### PR TITLE
build: implement config to sync event names

### PR DESCRIPTION
**Pull request by kyubisation**
## Changes

Changes in this pull request:

Implement config to sync event names from `@Event({...})` to `component`.events.ts file.

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Mentions

@lyne-design-system/core

### CODE REVIEW

File: src/components/lyne-button/lyne-button.tsx [hunk 1]
```diff
@@ -64,27 +64,12 @@ export class LyneButton {
...
-    });
-
-    this._element.dispatchEvent(event);
-  };
+  /** Emits whenever the native button click event triggers. */
+  @Event({
+    bubbles: true,
+    composed: true,
+    eventName: 'lyne-button_click'
+  }) public click: EventEmitter<any>;
```
Comment (jeripeierSBB) [inline src/components/lyne-button/lyne-button.tsx:72]: when using option visualButtonOnly, the event is not anymore prevented.

Comment (jeripeierSBB) [inline src/components/lyne-button/lyne-button.tsx:72]: Can we use another type than any?

Comment (kyubisation) [reply src/components/lyne-button/lyne-button.tsx:75]: Added todo, as it's not yet clear to me what the best type should be.

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**